### PR TITLE
Only close posting window when pasting succeeded

### PR DESCRIPTION
**Pull request by SummerWish**
#113

### CODE REVIEW

File: javascripts/background_scripts/posting_process.js [hunk 1]
```diff
@@ -107,18 +107,23 @@ var postingProcess = {
       postingProcess.postingResultTab !== undefined) {
 
       //Switches current tab to the page receiving the URL
-      chrome.tabs.update(postingProcess.postingResultTab.id, {selected: true});
+      // selected is deprecated
+      // See https://groups.google.com/a/chromium.org/forum/#!topic/chromium-extensions/gWamKdDzZNQ
+      // chrome.tabs.update(postingProcess.postingResultTab.id, {selected: true});
```
Comment (smcgregor) [inline javascripts/background_scripts/posting_process.js]: Try "highlighted".

https://developer.chrome.com/extensions/tabs#method-update

Comment (SummerWish) [inline javascripts/background_scripts/posting_process.js]: I found that it works properly without focuing the tab. So I just commented that line :-)

Comment (smcgregor) [inline javascripts/background_scripts/posting_process.js]: What about if someone initiates posting, finds content in another tab, then
completes the post? The user won't see the outcome of the posting process.

Comment (SummerWish) [inline javascripts/background_scripts/posting_process.js]: I use `active` now. `highlighted` doesn't work as expected (it makes two tabs highlighted).

**Comment by smcgregor** [comment]
@SummerWish Is this incorporated in your other pull requests? Regardless, I think it may be best to close this and implement on top of the messaging interface?

**Comment by SummerWish** [comment]
Yup. This PR can be replaced by the seamless-posting. Closed.

**Pull request closed without merging**